new-evaluation.html

```diff
@@ -0,0 +1,563 @@
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+							</li>
+						</ul>
+						<ul>
+							<li>Libraries</li>
+							<li>
+								<select name="libraries">
+									<option selected disabled hidden>Evaluation</option>
+									<option value="0">0</option>
+									<option value="1">1</option>
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+							</li>
+							<li>Promises</li>
+							<li>
+								<select name="promises">
+									<option selected disabled hidden>Evaluation</option>
+									<option value="0">0</option>
+									<option value="1">1</option>
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+							</li>
+							<li>Browser Engines</li>
+							<li>
+								<select name="browser">
+									<option selected disabled hidden>Evaluation</option>
+									<option value="0">0</option>
+									<option value="1">1</option>
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+							</li>
+						</ul>
+						<ul>
+						<li>Frameworks</li>
+						<li>
+							<select name="frameworks">
+									<option selected disabled hidden>Evaluation</option>
+									<option value="0">0</option>
+									<option value="1">1</option>
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+						</li>
+						</ul>
+					</fieldset>
+					<fieldset>
+						<legend>NodeJS</legend>
+						<ul>
+						<li>Backend frameworks</li>
+						<li>
+							<select name="backend">
+									<option selected disabled hidden>Evaluation</option>
+									<option value="0">0</option>
+									<option value="1">1</option>
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+						</li>
+						<li>Templating</li>
+						<li>
+							<select name="templating">
+									<option selected disabled hidden>Evaluation</option>
+									<option value="0">0</option>
+									<option value="1">1</option>
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+						</li>
+						<li>DOM Manipulation</li>
+						<li>
+							<select name="dom_manip">
+									<option selected disabled hidden>Evaluation</option>
+									<option value="0">0</option>
+									<option value="1">1</option>
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+						</li>
+						</ul>
+						<ul>
+						<li>Unit Testing</li>
+						<li>
+							<select name="unit_test">
+									<option selected disabled hidden>Evaluation</option>
+									<option value="0">0</option>
+									<option value="1">1</option>
+									<option value="2">2</option>
+									<option value="3">3</option>
+								</select>
+						</li>
+						</ul>
+					</fieldset>
+					<br>
+					<button id="submit" type="submit">SUBMIT</button>
```

Comment:
So, let's say that I'd want this text to be uppercase just on mobile, but proper capital case on desktop. Would you be able to build it that way?